data/scripting/messages.lua

```diff
@@ -41,7 +52,9 @@ end
 --    Sends a message to the inbox of all players and show it instantly.
 --    This is mainly used for winconditions to show the status.
 --
---    :arg text: the localized body of the message. You can use rt functions here.
+--    :arg text: the localized body of the message. You can use
+--               :ref:`richtext functions <richtext.lua>` here.
+--               E.g. ``p(_"text")``.
```

Comment:
```suggestion
--               E.g. ``p(_("text"))``.
```
Don't encourage people to leave out optional `()` ;)

Reply:
Done, also changed this in the code part